tests/datamodules/test_datamodules.py

```diff
@@ -112,7 +112,7 @@ def test_sr_datamodule(datadir):
 
 @pytest.mark.parametrize("split", ["byclass", "bymerge", "balanced", "letters", "digits", "mnist"])
 @pytest.mark.parametrize("dm_cls", [BinaryEMNISTDataModule, EMNISTDataModule])
```

Comment:
Can these tests also test `FashionMnistDataModule`? This applies to all the tests

Reply:
I don't think so. I think that these tests are only applicable to EMNIST and it's derivatives.

Reply:
Yes, you're right, sorry about that.